### PR TITLE
Fix Daily Build Failure

### PR DESCRIPTION
**Pull request by DimuthuMadushan**
## Purpose

Fixes: [#6965](https://github.com/ballerina-platform/ballerina-library/issues/6965)

## Checklist
- [x] Linked to an issue
- [ ] ~Updated the specification~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: .github/workflows/build-with-bal-test-graalvm.yml [hunk 1]
```diff
@@ -15,3 +15,5 @@ jobs:
     if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
     uses: ballerina-platform/ballerina-library/.github/workflows/build-with-bal-test-graalvm-connector-template.yml@main
     secrets: inherit
+    with:
```
Comment (ayeshLK) [inline .github/workflows/build-with-bal-test-graalvm.yml:18]: In the other connectors we disabled the examples build from the gradle build level. If we do that we don;t have to update all the places.

File: ballerina/tests/test.bal [hunk 2]
```diff
@@ -17,15 +17,11 @@
...
-configurable string & readonly refreshToken = os:getEnv("REFRESH_TOKEN");
-configurable string & readonly clientId = os:getEnv("CLIENT_ID");
-configurable string & readonly clientSecret = os:getEnv("CLIENT_SECRET");
-
 configurable string mockClientId = ?;
 configurable string mockClientSecret = ?;
 configurable string mockRefreshToken = ?;
 configurable string mockRefreshUrl = ?;
-configurable boolean isTestOnLiveServer = ?;
+configurable boolean isTestOnLiveServer = false;
```
Comment (ayeshLK) [inline ballerina/tests/test.bal:24]: I think this should be derived from an environment variable.

File: .github/workflows/build-with-bal-test-graalvm.yml (hunk 1 above)
Comment (DimuthuMadushan) [reply .github/workflows/build-with-bal-test-graalvm.yml:18]: In that case, example build will skip for local build as well. right?

File: ballerina/tests/test.bal (hunk 2 above)
Comment (DimuthuMadushan) [reply ballerina/tests/test.bal:24]: This is not added as env variable in the repo.

File: .github/workflows/build-with-bal-test-graalvm.yml (hunk 1 above)
Comment (NipunaRanasinghe) [reply .github/workflows/build-with-bal-test-graalvm.yml:18]: @DimuthuMadushan yes. that's the approach we followed for the other connectors as well. So lets follow the same approach ftm.

File: ballerina/types.bal [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
 import ballerina/http;
 
+// Update the config fields with default values due to lang issue https://github.com/ballerina-platform/ballerina-lang/issues/43359
```
Comment (NipunaRanasinghe) [inline ballerina/types.bal:19]:
```suggestion
// TODO: revert the changes once the lang issue https://github.com/ballerina-platform/ballerina-lang/issues/43359 is fixed in a 2201.10.x patch release
```

File: ballerina/tests/test.bal (hunk 2 above)
Comment (DimuthuMadushan) [reply ballerina/tests/test.bal]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.sheets/pull/297/commits/0c17b9697fd969f88dae2df7c1a43232255a88a3

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**